_unittest_solvers/test_26_emit.py

```diff
@@ -18,6 +18,7 @@
 from pyaedt.modeler.circuits.PrimitivesEmit import EmitComponents
 
 test_subfolder = "T26"
+test_review_flag = True
```

Comment:
I would suggest to rename the constants using upper case naming convention.
```suggestion
TEST_SUBFOLDER = "T26"
TEST_REVIEW_FLAG = True
```